app/services/order_details/reconciler.rb

```diff
@@ -12,8 +12,17 @@ class Reconciler
     validate :reconciliation_must_be_in_past
     validate :all_journals_and_statements_must_be_before_reconciliation_date
 
-    def initialize(order_detail_scope, params, reconciled_at, bulk_reconcile_note = nil, bulk_deposit_number = nil, bulk_reconcile_checkbox = nil)
+    def initialize(
+      order_detail_scope,
+      params,
+      reconciled_at,
+      bulk_reconcile_note = nil,
+      bulk_deposit_number = nil,
+      bulk_reconcile_checkbox = nil,
+      order_status = "reconciled"
+    )
```

Comment:
I like this - it's much easier to read this way.  I think it should be indented to the right a bit though?